src/Raven.Studio/typescript/models/resources/info/databaseInfo.ts

```diff
@@ -44,6 +44,34 @@ class databaseInfo extends resourceInfo {
         this.documentsCount(databaseInfo.DocumentsCount);
         this.indexesCount(databaseInfo.IndexesCount);
     }
+
+    static empty(resourceName: string): databaseInfo {
```

Comment:
We cannot get the information about the database until it's actually loaded anyway.
After we receive the "Load" notification we fetch the stats.